<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方法管理</string>
  </property>
  <widget class="QPushButton" name="pb_new">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_import">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save_as">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_action">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>应用并返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_method">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>581</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
